hasher-matcher-actioner/hmalib/indexers/lcc.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import pickle
+import time
+import typing as t
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+
+from threatexchange.signal_type.pdq_index import PDQIndex
+
+from hmalib.common.logging import get_logger
+from hmalib.common.models.pipeline import HashRecord
+from hmalib.common.timebucketizer import TimeBucketizer
+
+starttime = time.time()
+logger = get_logger(__name__)
+
+# parameters of get_records
+# def get_records(
+#         since: datetime.datetime,
+# until: datetime.datetime,
+# type: str,
+# storage_path: str,
+# bucket_width: datetime.timedelta,
+# type_class: t.Type[CSViable],
+# ):
+
+
+class LCCIndexer:
+    @classmethod
+    def get_recent_index(cls) -> PDQIndex:
+        """Get the most recent index."""
+
+    # find some way to access most recent index in file structure
+    # indexObjPath = path of index with latest creation time (should already be sorted by time in file structure)
+    # file = open(indexObjPath,"r")
+    # latest_index = pickle.load(file)
+    # file.close()
+    # return latest_index
+
+    @classmethod
+    def build_index_from_last_24h(cls, tbi) -> None:
```

Comment:
Instead of receiving the tbi instance which you don't use at all, receive type, storage_path and bucket_width as arguments.

Reply:
build_index_from_last_24h returns SignalTypeIndiex. Fix that in the mypy types.